docs/references/javascript/clerk/session-methods.mdx

```diff
@@ -5,47 +5,12 @@ description: These are all methods on the Clerk class that help you manage the a
 
 # Session methods
 
-These are all methods on [the Clerk class](/docs/references/javascript/clerk/clerk) that help you manage the active session and/or organization.
-
-## `setSession()` (deprecated)
-
-```typescript
-function setSession(
-  session: Session | string | null,
-  beforeEmit?: BeforeEmitCallback
-): Promise<void>;
-```
-
-<Callout type="danger">
-  Deprecated in favor of [`setActive()`](#set-active).
-</Callout>
-
-Set the current session on this client to the provided session. The provided session can be either a complete [`Session`][session-ref] object or simply its unique identifier.
-
-Passing null as the session will result in the current session to be removed from the client.
-
-If an active session already exists, it will be replaced with the new one. The change happens in three steps:
-
-1. The current [`Session`][session-ref] object is set to `undefined`, which causes the control components to stop rendering their children as though Clerk is still initializing.
-2. The `beforeEmit` callback is executed. If a `Promise` is returned, Clerk waits for the `Promise` to resolve.
-3. The current [`Session`][session-ref] is set to the passed `session`. This causes the control components to render their children again.
-
-### Properties
-
-
-| Name | Type | Description |
-| --- | --- | --- |
-| `session` | [`Session`][session-ref] \| `string` \| `null` | A `Session` object or `Session ID` string to be set as the current session, or `null` to simply remove the active session, without setting a new one. |
-| `beforeEmit?` | `(session: Session \| null) => Promise<any>` | Callback that will trigger when the current session is set to `undefined`, before finally being set to the passed session. Usually used for navigation. |
-
-### Returns
-
-| Type | Description |
-| --- | --- |
-| `Promise<void>` | The `Promise` will resolve after the passed `session` is set. |
+These are all methods on the [`Clerk`](/docs/references/javascript/clerk/clerk) class that help you manage the active session and/or organization.
 
 ## `setActive()`
 
+A method used to set the active session and/or organization.
```

Comment:
> A method used to set the active session and/or organization.

Suggestion 1: A method used to set the active session and organization, or both.

Suggestion 2: A method used to set the active session or organization.